files/docker-container-logrotate

```diff
@@ -0,0 +1,9 @@
+/var/lib/docker/containers/*/*.log {
+  rotate 0
+  hourly
+  notifempty
+  nocompress
+  size 20k
```

Comment:
We can Well, right now, the logrotate service gets called at 24 hours cadence. Which means